src/shared/components/schoolCard/schoolCard.js

```diff
@@ -1,52 +1,51 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import styles from './schoolCard.css';
 
-const SchoolCard = ({
-  alt,
-  GI,
-  fullTime,
-  hardware,
-  link,
-  logo,
-  schoolAddress,
-  schoolCity,
-  schoolName,
-  schoolState
-}) => (
-  <div className={styles.schoolCard}>
-    <div className={styles.schoolCardImage}>
-      <a href={link} target="_blank" rel="noopener noreferrer">
-        <img src={logo} alt={alt} className={styles.logo} />
-      </a>
-    </div>
+class SchoolCard extends Component {
```

Comment:
Can you convert this back into a functional component?

`const SchoolCard = ({ destructuredProps }) => { ... };`